golang/cmd/mqtt-to-postgresql/database.go

```diff
@@ -1909,3 +1999,28 @@ func GetComponentID(assetID int, componentName string) (componentID int) {
 
 	return
 }
+
+// Todo (naming conventions), sachen
+func GetUniqueProductID(aid int, assetID int) (uid int, err error) {
```

Comment:
This function is perfectly suited for caching. When calling it, always check the cache beforehand (you can find the cache in the internal package `golang/PKG/internal/cache.go`)

Reply:
Good idea, thank you! I implemented a uniqueProductID cache (`GetUniqueProductIDFromCache()`, `StoreUniqueProductIDToCache()`).